src/AddIns/Uno.UI.Lottie.Wasm/WasmScripts/uno-lottie.d.ts

```diff
@@ -0,0 +1,36 @@
+declare const require: any;
+declare const config: any;
+declare namespace Uno.UI {
+    import AnimationData = Lottie.AnimationData;
+    interface LottieAnimationProperties {
+        elementId: number;
+        jsonPath?: string;
+        autoplay: boolean;
+        stretch: string;
+        rate: number;
+    }
+    interface RunningLottieAnimation {
+        animation: Lottie.AnimationItem;
+        properties: LottieAnimationProperties;
+    }
+    class Lottie {
+        private static _player;
+        private static _runningAnimations;
+        private static _numberOfFrames;
+        static setAnimationProperties(newProperties: LottieAnimationProperties, animationData?: AnimationData): string;
+        static stop(elementId: number): string;
+        static play(elementId: number, fromProgress: number, toProgress: number, looped: boolean): string;
+        static kill(elementId: number): string;
+        static pause(elementId: number): string;
```

Comment:
Codacy found an issue: [The class method 'pause' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)